cmd/timescale-prometheus/main.go

```diff
@@ -329,6 +340,54 @@ func parseFlags() (*config, error) {
 	return cfg, nil
 }
 
+var errTimedOut = fmt.Errorf("timed out waiting to connect to DB")
+
+func waitForDb(cfg *config) error {
+	maxTimeout := 60 * time.Second
```

Comment:
This should be a cli flag and we need to make sure that a value of 0 turns off the probe.

Reply:
Yes, and 60 secs be the default value.